spec/features/admin_dashboard_spec.rb

```diff
@@ -27,6 +27,7 @@ def login
       within(".uncontacted-companies") do
         expect(page).to have_content(companies.first.organization)
         expect(page).to have_content(companies.last.organization)
+        expect(companies.first.updated_at <= companies.last.updated_at).to eq(true)
```

Comment:
At first I thought this was a reverse hashrocket. Good way to check that the list appears in the right order!